### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gitatmax**

### CODE REVIEW

**Comment by nikivdev** [comment]
The headings shouldn't be changed. The other changes seem fine.

**Comment by nikivdev** [comment]
Just realized you are the person behind Pacmax, that's awesome.

Just to clarfiy, the headings shouldn't be changed as I want it consistent with [other lists](https://github.com/learn-anything/curated-lists). So best leave it as it is, I don't think it's an issue.

**Comment by gitatmax** [comment]
Hi! How does that look? Please let me know if I should make any other adjustments. 

Love the list, and Learn Anything!

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ _Please read [contribution guidelines](CONTRIBUTING.md#readme) before contributi
 - [Writing](#writing)
 - [Meta](#meta)
 - [Related](#related)
+- [GitHub badge](#github-badge)
```
Comment (nikivdev) [inline README.md]: No need for this, it's not as important to jump to it

**Review by nikivdev — CHANGES_REQUESTED** [review]

**Comment by nikivdev** [comment]
One comment then I can merge

**Comment by nikivdev** [comment]
Thanks

**Comment by gitatmax** [comment]
You're welcome

**Pull request merged**